_posts/2018-07-17-meetup.md

```diff
@@ -0,0 +1,46 @@
+---
+title: 
+author: lon
+layout: post
+categories:
+- posts
+- meetups
+when: 2018-07-17T19:30:00-05:00
+---
+
+{% assign speakr = 'Dave Rupert' %}
+{% assign twiturl = 'https://twitter.com/davatron5000' %}
+{% assign huburl = 'https://github.com/davatron5000/' %}
+
+### Zelda + a11y?
+
+Accessibility is hard. Oftentimes, it’s difficult to understand what users and
+assistive technology expect from your markup and designs. Can accessibility be
+easier to learn? Can it be…fun? In this talk, <a href="{{ twiturl }}">{{ speakr
```

Comment:
`&hellp;`? dunno if the jekyll machines in the GitHub™️ (a Microsoft™️ company) will process this correctly.